### ISSUE TITLE
Lsd long output is not properly aligned

### ISSUE THREAD

**Issue by pwnalone** [lsd-rs/lsd#1063]
- os: `Linux kali 6.6.15-amd64 #1 SMP PREEMPT_DYNAMIC Kali 6.6.15-2kali1 (2024-05-17) x86_64 GNU/Linux` (also observed on Ubuntu under WSL)
- `lsd --version`: 1.1.2 (also observed with 1.0.0)
- `echo $TERM`: `tmux-256color` (also observed with `alacritty`)
- `echo $LS_COLORS`: (empty)

## Expected behavior

The output should be properly aligned as it is with vanilla `ls`.

![image](https://github.com/lsd-rs/lsd/assets/3609458/4e1bd588-afb2-4dd5-9120-9e33944078f1)

## Actual behavior

The output is not aligned properly.

![image](https://github.com/lsd-rs/lsd/assets/3609458/6072b9c2-58de-44c1-a231-98ac85558db8)

**Comment by ehwan** [comment]
I'm having same issue on OSX Terminal.app

But it worked good when I put `--color=never`.

![image](https://github.com/lsd-rs/lsd/assets/15500385/7b136a71-6a5e-4842-92fd-d882e9416a85)

![image](https://github.com/lsd-rs/lsd/assets/15500385/80c40abe-2d18-4484-a6d2-686b4b47a06b)

**Comment by zwpaper** [comment]
is there any chance I can reproduce this?

**Comment by thomcc** [comment]
I have this bug too, it reproduces easily macOS and Linux on every folder I've tried.

**Comment by efvincent** [comment]
same issue. version 1.1.2, installed today with 
```bash
cargo install --git https://github.com/lsd-rs/lsd.git --branch master
```
KDE Neon (Ubuntu) 6.0
Plasma 6.1.2
KDE Framework 6.3.0
Qt 6.7
Kernel 6.5.0-41-generic 64bit
Wayland

terminal: Kitty 0.21.2
font: Fira Code Nerd Font Mono Regular

**Comment by msrd0** [comment]
@efvincent Add `--locked` to your command and the issue will go away

**Comment by efvincent** [comment]
> @efvincent Add `--locked` to your command and the issue will go away

That did it... didn't notice the related issue that had the answer - thanks for pointing it out! 👍🏽

**Comment by SamuelLarkin** [comment]
Hi,
  just to add another data point.  In my case I get the directories to be misaligned with the files or is it the other way around 😕 

```sh
lsd --version
lsd 1.1.2
```

```sh
❯ lsd
.rw-r--r--    larkins larkins  10 KB Fri Jul  5 10:49:45 2024  dot_alias
.rw-rw----    larkins larkins 4.5 KB Tue Jul 16 12:15:09 2024  dot_bashrc.common
drwxr-xr-x larkins larkins 4.0 KB Fri Jul  5 10:49:45 2024  dot_cargo
drwxr-xr-x larkins larkins 4.0 KB Fri Jul  5 10:49:45 2024  dot_config
.rw-r--r--    larkins larkins  19 KB Tue Jul 16 12:11:09 2024  dot_gdbinit
.rw-r--r--    larkins larkins 8.3 KB Tue Jul 16 12:12:17 2024  dot_gitconfig
.rw-r--r--    larkins larkins 910 B  Fri Jul  5 10:49:45 2024  dot_gotz.config.json
.rw-r--r--    larkins larkins  52 B  Fri Jul  5 10:49:45 2024  dot_inputrc
.rw-r--r--    larkins larkins 1.3 KB Fri Jul  5 10:49:45 2024  dot_screenrc
.rw-r--r--    larkins larkins 125 B  Fri Jul  5 10:49:45 2024  dot_tigrc
.rw-r--r--    larkins larkins  13 KB Tue Jul 16 12:02:48 2024  dot_tmux.conf
.rw-r--r--    larkins larkins 967 B  Fri Jul  5 10:49:45 2024  dot_toprc
.rw-rw----    larkins larkins  17 KB Tue Jul 16 12:19:56 2024  dot_vimrc
.rw-rw----    larkins larkins 409 B  Fri Jul  5 15:38:28 2024  README.md
```

I recompiled the same version with `--locked` and the issue went away

```sh
❯ lsd
.rw-r--r-- larkins larkins  10 KB Fri Jul  5 10:49:45 2024  dot_alias
.rw-rw---- larkins larkins 4.5 KB Tue Jul 16 12:15:09 2024  dot_bashrc.common
drwxr-xr-x larkins larkins 4.0 KB Fri Jul  5 10:49:45 2024  dot_cargo
drwxr-xr-x larkins larkins 4.0 KB Fri Jul  5 10:49:45 2024  dot_config
.rw-r--r-- larkins larkins  19 KB Tue Jul 16 12:11:09 2024  dot_gdbinit
.rw-r--r-- larkins larkins 8.3 KB Tue Jul 16 12:12:17 2024  dot_gitconfig
.rw-r--r-- larkins larkins 910 B  Fri Jul  5 10:49:45 2024  dot_gotz.config.json
.rw-r--r-- larkins larkins  52 B  Fri Jul  5 10:49:45 2024  dot_inputrc
.rw-r--r-- larkins larkins 1.3 KB Fri Jul  5 10:49:45 2024  dot_screenrc
.rw-r--r-- larkins larkins 125 B  Fri Jul  5 10:49:45 2024  dot_tigrc
.rw-r--r-- larkins larkins  13 KB Tue Jul 16 12:02:48 2024  dot_tmux.conf
.rw-r--r-- larkins larkins 967 B  Fri Jul  5 10:49:45 2024  dot_toprc
.rw-rw---- larkins larkins  17 KB Tue Jul 16 12:19:56 2024  dot_vimrc
.rw-rw---- larkins larkins 409 B  Fri Jul  5 15:38:28 2024  README.md
```

**Comment by pj-connect** [comment]
Same issue here, as describes [ehwan](https://github.com/ehwan), in kitty 0.35.0 on Ubuntu 22.04. `--locked` not working. Built from source : lsd 1.1.3.


```
> .cargo/bin/lsd --locked
error: unexpected argument '--locked' found

  tip: a similar argument exists: '--blocks'

Usage: lsd <FILE|--all|--almost-all|--color <MODE>|--icon <MODE>|--icon-theme <THEME>|--classify|--long|--ignore-config|--config-file <PATH>|--oneline|--recursive|--human-readable|--tree|--depth <NUM>|--directory-only|--permission <MODE>|--size <MODE>|--total-size|--date <DATE>|--timesort|--sizesort|--extensionsort|--gitsort|--versionsort|--sort <TYPE>|--no-sort|--reverse|--group-dirs <MODE>|--group-directories-first|--blocks <BLOCKS>|--classic|--no-symlink|--ignore-glob <PATTERN>|--inode|--git|--dereference|--context|--hyperlink <MODE>|--header|--truncate-owner-after <NUM>|--truncate-owner-marker <STR>|--system-protected|--literal|--help>
```

**Comment by msrd0** [comment]
@pj-connect --locked is a compile time flag, not a runtime flag

**Comment by zwpaper** [comment]
@pj-connect can you try the binary on github release page

**Comment by zwpaper** [comment]
I have merged https://github.com/lsd-rs/lsd/pull/1064, let's see if v1.1.5 helps

**Comment by nc7s** [comment]
FWIW, this might be https://github.com/unicode-rs/unicode-width/issues/55. Adding `--locked` uses its "locked" version, 0.1.11, which is before the problematic 0.1.13.

**Comment by zwpaper** [comment]
this should now be fixed in 1.1.5
/close
